### PR TITLE
Scripts/Kalimdor: Move script of Hand of Iruxos Crystal to database.

### PR DESCRIPTION
**Pull request by Lumber-Jack-2**
**Changes proposed:**

-  Move script of Hand of Iruxos Crystal to database.
-  
-  

**Target branch(es):** 3.3.5/master

- [X] 3.3.5
- [ ] master

**Issues addressed:** Closes #  (insert issue tracker number)

**Tests performed:** (Does it build, tested in-game, etc.)
- Yes, all works

**Known issues and TODO list:** (add/remove lines as needed)
- Nothing

### CODE REVIEW

**Comment by Killyana** [comment]
The gob despawns once open, in this case no need to for the SAI condition I think.

**Comment by Rushor** [comment]
ah actually it does not despawn for some reason :o (was like that before already)

**Comment by Rushor** [comment]
despawn no is ok

**Comment by Killyana** [comment]
Change the event and the respawn time, and it's ok.
``` sql
UPDATE `gameobject` SET `spawntimesecs`=180 WHERE `id`=176581;
SET @ENTRY := 176581;
UPDATE `gameobject_template` SET `AIName`="SmartGameObjectAI", `scriptname`= '' WHERE `entry`=@ENTRY;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=1;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,1,0,1,70,0,100,0,2,0,0,0,12,11876,4,10000,0,0,0,1,0,0,0,0,0,0,0,"Hand of Iruxos Crystal - On gob state activated - Summon Creature 'Demon Spirit' for 10000 milliseconds "),
(@ENTRY,1,1,0,61,0,100,0,0,0,0,0,41,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Hand of Iruxos Crystal - On gob state activated - Despawn ");

**Comment by Rushor** [comment]
aye

File: src/server/scripts/Kalimdor/zone_desolace.cpp [hunk 1]
```diff
@@ -25,8 +25,6 @@ EndScriptData */
 /* ContentData
 npc_aged_dying_ancient_kodo
 go_iruxos
```
Comment (illfated) [inline src/server/scripts/Kalimdor/zone_desolace.cpp]:
```suggestion
```
This line will effectively also be redundant when its script is moved to SAI.

Comment (Rushor) [reply src/server/scripts/Kalimdor/zone_desolace.cpp]: that's why i removed it

Comment (illfated) [reply src/server/scripts/Kalimdor/zone_desolace.cpp]: Yes, but I suggest removing this comment line too.

Comment (illfated) [reply src/server/scripts/Kalimdor/zone_desolace.cpp]: Thanks. You can now mark this conversation as resolved, to minimize it. :)

**Pull request merged**